src/desktop/src/ui/views/wallet/Receive.js

```diff
@@ -128,7 +128,10 @@ class Receive extends React.PureComponent {
                     >
                         <p>
                             {receiveAddress.split('').map((char, index) => {
-                                const scrambleChar = scramble[index] > 0 ? byteToChar(scramble[index]) : null;
+                                const scrambleChar =
+                                    scramble[index] > 0
+                                        ? '9ABCDEFGHIJKLMNOPQRSTUVWXYZ'.charAt(scramble[index] % 27)
```

Comment:
Might be easier if we have this charset stored in another file

Reply:
We actually, do it's `charToByte` function at `libs/helpers`. Will update to use that.